test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -445,17 +450,24 @@ def setUp(self):
             return
         self.log_enter_setup()
         self.init_proc_mon()
-        _, path = tempfile.mkstemp(prefix="setup_output",
-                                   suffix=".json")
-        self.saved_file = path
-        self.server.save_configuration(path, 'w')
+        if not PBSTestSuite.config_saved and self.use_cur_setup:
+            _, path = tempfile.mkstemp(prefix="saved_test_setup",
+                                       suffix=".json")
+            self.saved_test_setup = self.server.save_configuration(path, 'w')
+            if self.saved_test_setup:
+                self.saved_file = path
+            else:
+                self.logger.error("Failed to save test setup")
+                PBSTestSuite.config_saved = True
```

Comment:
I think we should stop executing and error out if the save fails, what do you think?

Reply:
Stopping the execution when a command fails.